### PR TITLE
Add inventory script for ansible-playbook

### PR DESCRIPTION
**Pull request by keedya**
- Allow ansible to get hosts inventory from rackHD by node id
- the ansible job looks for this script in /opt/onrack/etc/ansible/

### CODE REVIEW

**Comment by keedya** [comment]
@zyoung51 @tannoa2 @uppalk1 @jlongever @RackHD/corecommitters

**Comment by zyoung51** [comment]
+1

File: scripts/inventory.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+import json
+import requests
+import os
+import argparse
+import types
+
+MONORAIL_URL = 'http://localhost:8080'
+
+class OnRackInventory(object):
```
Comment (jlongever) [inline scripts/inventory.py]: nit:  rename to rackhd

File: scripts/inventory.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+import json
+import requests
+import os
+import argparse
+import types
+
+MONORAIL_URL = 'http://localhost:8080'
```
Comment (jlongever) [inline scripts/inventory.py]: nit: RACKHD_URL

File: scripts/inventory.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            output += '      "ansible_ssh_user": "renasar"\n'
+            output += '    }\n'
+            output += '  }\n'
+        except KeyError:
+            pass
+        return output
+
+try:
+    #check if monorail url(ie:10.1.1.45:8080) is specified in the environment
+    MONORAIL_URL = 'http://' + str(os.environ['MONORAIL'])
```
Comment (jlongever) [inline scripts/inventory.py]: nit: define env as RACKHD_URL

File: scripts/inventory.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+import json
+import requests
+import os
+import argparse
+import types
+
+<<<<<<< HEAD
```
Comment (benbp) [inline scripts/inventory.py]: You've got rebase diff conflict lines in here.

**Pull request closed without merging**